{"message":"Jun  4 14:06:19 bd-3-12 sudo: pam_unix(sudo:session): session opened for user root by (uid=110)","@version":"1","@timestamp":"2014-06-04T21:06:19.526Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:19 bd-3-12 sudo: pam_unix(sudo:session): session closed for user root","@version":"1","@timestamp":"2014-06-04T21:06:19.526Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:19 bd-3-12 sudo:  neutron : TTY=unknown ; PWD=/ ; USER=root ; COMMAND=/usr/bin/neutron-rootwrap /etc/neutron/rootwrap.conf ovs-vsctl --timeout=2 --format=json -- --columns=name,external_ids list Interface","@version":"1","@timestamp":"2014-06-04T21:06:19.526Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:19 bd-3-12 sudo: pam_unix(sudo:session): session opened for user root by (uid=110)","@version":"1","@timestamp":"2014-06-04T21:06:19.526Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:19 bd-3-12 sudo: pam_unix(sudo:session): session closed for user root","@version":"1","@timestamp":"2014-06-04T21:06:19.526Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:20 bd-3-12 sshd[5224]: Accepted publickey for ubuntu from 10.0.130.103 port 49868 ssh2","@version":"1","@timestamp":"2014-06-04T21:06:21.530Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:20 bd-3-12 sshd[5224]: pam_unix(sshd:session): session opened for user ubuntu by (uid=0)","@version":"1","@timestamp":"2014-06-04T21:06:21.531Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:21 bd-3-12 sudo:  neutron : TTY=unknown ; PWD=/ ; USER=root ; COMMAND=/usr/bin/neutron-rootwrap /etc/neutron/rootwrap.conf ovs-vsctl --timeout=2 list-ports br-int","@version":"1","@timestamp":"2014-06-04T21:06:21.531Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:21 bd-3-12 sudo: pam_unix(sudo:session): session opened for user root by (uid=110)","@version":"1","@timestamp":"2014-06-04T21:06:21.531Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:21 bd-3-12 sshd[5372]: Received disconnect from 10.0.130.103: 11: disconnected by user","@version":"1","@timestamp":"2014-06-04T21:06:21.531Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:21 bd-3-12 sshd[5224]: pam_unix(sshd:session): session closed for user ubuntu","@version":"1","@timestamp":"2014-06-04T21:06:21.531Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:21 bd-3-12 sudo: pam_unix(sudo:session): session closed for user root","@version":"1","@timestamp":"2014-06-04T21:06:21.531Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:21 bd-3-12 sudo:  neutron : TTY=unknown ; PWD=/ ; USER=root ; COMMAND=/usr/bin/neutron-rootwrap /etc/neutron/rootwrap.conf ovs-vsctl --timeout=2 --format=json -- --columns=name,external_ids list Interface","@version":"1","@timestamp":"2014-06-04T21:06:21.531Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:21 bd-3-12 sudo: pam_unix(sudo:session): session opened for user root by (uid=110)","@version":"1","@timestamp":"2014-06-04T21:06:21.531Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:21 bd-3-12 sudo: pam_unix(sudo:session): session closed for user root","@version":"1","@timestamp":"2014-06-04T21:06:21.531Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"2014-06-04 14:06:21.716 18971 WARNING ceilometer.compute.pollsters.bandwidth [-] Ignoring instance instance-00000d1d: Can't reconnect until invalid transaction is rolled back (original cause: InvalidRequestError: Can't reconnect until invalid transaction is rolled back) 'SELECT ports.tenant_id AS ports_tenant_id, ports.id AS ports_id, ports.name AS ports_name, ports.network_id AS ports_network_id, ports.mac_address AS ports_mac_address, ports.admin_state_up AS ports_admin_state_up, ports.status AS ports_status, ports.device_id AS ports_device_id, ports.device_owner AS ports_device_owner \\nFROM ports' [immutabledict({})]","@version":"1","@timestamp":"2014-06-04T21:06:22.534Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 ERROR ceilometer.compute.pollsters.bandwidth [-] Can't reconnect until invalid transaction is rolled back (original cause: InvalidRequestError: Can't reconnect until invalid transaction is rolled back) 'SELECT ports.tenant_id AS ports_tenant_id, ports.id AS ports_id, ports.name AS ports_name, ports.network_id AS ports_network_id, ports.mac_address AS ports_mac_address, ports.admin_state_up AS ports_admin_state_up, ports.status AS ports_status, ports.device_id AS ports_device_id, ports.device_owner AS ports_device_owner \\nFROM ports' [immutabledict({})]","@version":"1","@timestamp":"2014-06-04T21:06:22.534Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 TRACE ceilometer.compute.pollsters.bandwidth Traceback (most recent call last):","@version":"1","@timestamp":"2014-06-04T21:06:22.534Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 TRACE ceilometer.compute.pollsters.bandwidth   File \"/usr/local/lib/python2.7/dist-packages/ceilometer/compute/pollsters/bandwidth.py\", line 513, in get_samples","@version":"1","@timestamp":"2014-06-04T21:06:22.534Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 TRACE ceilometer.compute.pollsters.bandwidth     yield self._get_sample(instance, vnic, info)","@version":"1","@timestamp":"2014-06-04T21:06:22.534Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 TRACE ceilometer.compute.pollsters.bandwidth   File \"/usr/local/lib/python2.7/dist-packages/ceilometer/compute/pollsters/bandwidth.py\", line 586, in _get_sample","@version":"1","@timestamp":"2014-06-04T21:06:22.534Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 TRACE ceilometer.compute.pollsters.bandwidth     PreConfigure()","@version":"1","@timestamp":"2014-06-04T21:06:22.534Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 TRACE ceilometer.compute.pollsters.bandwidth   File \"/usr/local/lib/python2.7/dist-packages/ceilometer/compute/pollsters/bandwidth.py\", line 109, in PreConfigure","@version":"1","@timestamp":"2014-06-04T21:06:22.534Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 TRACE ceilometer.compute.pollsters.bandwidth     neutron_ports = db.ports.all()","@version":"1","@timestamp":"2014-06-04T21:06:22.534Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 TRACE ceilometer.compute.pollsters.bandwidth   File \"/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/query.py\", line 2115, in all","@version":"1","@timestamp":"2014-06-04T21:06:22.534Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 TRACE ceilometer.compute.pollsters.bandwidth     return list(self)","@version":"1","@timestamp":"2014-06-04T21:06:22.534Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 TRACE ceilometer.compute.pollsters.bandwidth   File \"/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/query.py\", line 2227, in __iter__","@version":"1","@timestamp":"2014-06-04T21:06:22.534Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 TRACE ceilometer.compute.pollsters.bandwidth     return self._execute_and_instances(context)","@version":"1","@timestamp":"2014-06-04T21:06:22.534Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 TRACE ceilometer.compute.pollsters.bandwidth   File \"/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/query.py\", line 2242, in _execute_and_instances","@version":"1","@timestamp":"2014-06-04T21:06:22.534Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 TRACE ceilometer.compute.pollsters.bandwidth     result = conn.execute(querycontext.statement, self._params)","@version":"1","@timestamp":"2014-06-04T21:06:22.534Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 TRACE ceilometer.compute.pollsters.bandwidth   File \"/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py\", line 1449, in execute","@version":"1","@timestamp":"2014-06-04T21:06:22.534Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 TRACE ceilometer.compute.pollsters.bandwidth     params)","@version":"1","@timestamp":"2014-06-04T21:06:22.534Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 TRACE ceilometer.compute.pollsters.bandwidth   File \"/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py\", line 1584, in _execute_clauseelement","@version":"1","@timestamp":"2014-06-04T21:06:22.534Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 TRACE ceilometer.compute.pollsters.bandwidth     compiled_sql, distilled_params","@version":"1","@timestamp":"2014-06-04T21:06:22.534Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 TRACE ceilometer.compute.pollsters.bandwidth   File \"/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py\", line 1651, in _execute_context","@version":"1","@timestamp":"2014-06-04T21:06:22.534Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 TRACE ceilometer.compute.pollsters.bandwidth     None, None)","@version":"1","@timestamp":"2014-06-04T21:06:22.534Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 TRACE ceilometer.compute.pollsters.bandwidth   File \"/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py\", line 1645, in _execute_context","@version":"1","@timestamp":"2014-06-04T21:06:22.534Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 TRACE ceilometer.compute.pollsters.bandwidth     conn = self._revalidate_connection()","@version":"1","@timestamp":"2014-06-04T21:06:22.534Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 TRACE ceilometer.compute.pollsters.bandwidth   File \"/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py\", line 1044, in _revalidate_connection","@version":"1","@timestamp":"2014-06-04T21:06:22.535Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 TRACE ceilometer.compute.pollsters.bandwidth     \"Can't reconnect until invalid \"","@version":"1","@timestamp":"2014-06-04T21:06:22.535Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 TRACE ceilometer.compute.pollsters.bandwidth StatementError: Can't reconnect until invalid transaction is rolled back (original cause: InvalidRequestError: Can't reconnect until invalid transaction is rolled back) 'SELECT ports.tenant_id AS ports_tenant_id, ports.id AS ports_id, ports.name AS ports_name, ports.network_id AS ports_network_id, ports.mac_address AS ports_mac_address, ports.admin_state_up AS ports_admin_state_up, ports.status AS ports_status, ports.device_id AS ports_device_id, ports.device_owner AS ports_device_owner \\nFROM ports' [immutabledict({})]","@version":"1","@timestamp":"2014-06-04T21:06:22.535Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.717 18971 TRACE ceilometer.compute.pollsters.bandwidth ","@version":"1","@timestamp":"2014-06-04T21:06:22.535Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 WARNING ceilometer.compute.pollsters.bandwidth [-] Ignoring instance instance-00000ca0: Can't reconnect until invalid transaction is rolled back (original cause: InvalidRequestError: Can't reconnect until invalid transaction is rolled back) 'SELECT ports.tenant_id AS ports_tenant_id, ports.id AS ports_id, ports.name AS ports_name, ports.network_id AS ports_network_id, ports.mac_address AS ports_mac_address, ports.admin_state_up AS ports_admin_state_up, ports.status AS ports_status, ports.device_id AS ports_device_id, ports.device_owner AS ports_device_owner \\nFROM ports' [immutabledict({})]","@version":"1","@timestamp":"2014-06-04T21:06:22.535Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 ERROR ceilometer.compute.pollsters.bandwidth [-] Can't reconnect until invalid transaction is rolled back (original cause: InvalidRequestError: Can't reconnect until invalid transaction is rolled back) 'SELECT ports.tenant_id AS ports_tenant_id, ports.id AS ports_id, ports.name AS ports_name, ports.network_id AS ports_network_id, ports.mac_address AS ports_mac_address, ports.admin_state_up AS ports_admin_state_up, ports.status AS ports_status, ports.device_id AS ports_device_id, ports.device_owner AS ports_device_owner \\nFROM ports' [immutabledict({})]","@version":"1","@timestamp":"2014-06-04T21:06:22.535Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 TRACE ceilometer.compute.pollsters.bandwidth Traceback (most recent call last):","@version":"1","@timestamp":"2014-06-04T21:06:22.535Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 TRACE ceilometer.compute.pollsters.bandwidth   File \"/usr/local/lib/python2.7/dist-packages/ceilometer/compute/pollsters/bandwidth.py\", line 513, in get_samples","@version":"1","@timestamp":"2014-06-04T21:06:22.535Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 TRACE ceilometer.compute.pollsters.bandwidth     yield self._get_sample(instance, vnic, info)","@version":"1","@timestamp":"2014-06-04T21:06:22.535Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 TRACE ceilometer.compute.pollsters.bandwidth   File \"/usr/local/lib/python2.7/dist-packages/ceilometer/compute/pollsters/bandwidth.py\", line 586, in _get_sample","@version":"1","@timestamp":"2014-06-04T21:06:22.535Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 TRACE ceilometer.compute.pollsters.bandwidth     PreConfigure()","@version":"1","@timestamp":"2014-06-04T21:06:22.535Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 TRACE ceilometer.compute.pollsters.bandwidth   File \"/usr/local/lib/python2.7/dist-packages/ceilometer/compute/pollsters/bandwidth.py\", line 109, in PreConfigure","@version":"1","@timestamp":"2014-06-04T21:06:22.535Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 TRACE ceilometer.compute.pollsters.bandwidth     neutron_ports = db.ports.all()","@version":"1","@timestamp":"2014-06-04T21:06:22.535Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 TRACE ceilometer.compute.pollsters.bandwidth   File \"/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/query.py\", line 2115, in all","@version":"1","@timestamp":"2014-06-04T21:06:22.536Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 TRACE ceilometer.compute.pollsters.bandwidth     return list(self)","@version":"1","@timestamp":"2014-06-04T21:06:22.536Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 TRACE ceilometer.compute.pollsters.bandwidth   File \"/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/query.py\", line 2227, in __iter__","@version":"1","@timestamp":"2014-06-04T21:06:22.536Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 TRACE ceilometer.compute.pollsters.bandwidth     return self._execute_and_instances(context)","@version":"1","@timestamp":"2014-06-04T21:06:22.536Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 TRACE ceilometer.compute.pollsters.bandwidth   File \"/usr/local/lib/python2.7/dist-packages/sqlalchemy/orm/query.py\", line 2242, in _execute_and_instances","@version":"1","@timestamp":"2014-06-04T21:06:22.536Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 TRACE ceilometer.compute.pollsters.bandwidth     result = conn.execute(querycontext.statement, self._params)","@version":"1","@timestamp":"2014-06-04T21:06:22.536Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 TRACE ceilometer.compute.pollsters.bandwidth   File \"/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py\", line 1449, in execute","@version":"1","@timestamp":"2014-06-04T21:06:22.536Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 TRACE ceilometer.compute.pollsters.bandwidth     params)","@version":"1","@timestamp":"2014-06-04T21:06:22.536Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 TRACE ceilometer.compute.pollsters.bandwidth   File \"/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py\", line 1584, in _execute_clauseelement","@version":"1","@timestamp":"2014-06-04T21:06:22.536Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 TRACE ceilometer.compute.pollsters.bandwidth     compiled_sql, distilled_params","@version":"1","@timestamp":"2014-06-04T21:06:22.536Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 TRACE ceilometer.compute.pollsters.bandwidth   File \"/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py\", line 1651, in _execute_context","@version":"1","@timestamp":"2014-06-04T21:06:22.536Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 TRACE ceilometer.compute.pollsters.bandwidth     None, None)","@version":"1","@timestamp":"2014-06-04T21:06:22.537Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 TRACE ceilometer.compute.pollsters.bandwidth   File \"/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py\", line 1645, in _execute_context","@version":"1","@timestamp":"2014-06-04T21:06:22.537Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 TRACE ceilometer.compute.pollsters.bandwidth     conn = self._revalidate_connection()","@version":"1","@timestamp":"2014-06-04T21:06:22.537Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 TRACE ceilometer.compute.pollsters.bandwidth   File \"/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py\", line 1044, in _revalidate_connection","@version":"1","@timestamp":"2014-06-04T21:06:22.537Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 TRACE ceilometer.compute.pollsters.bandwidth     \"Can't reconnect until invalid \"","@version":"1","@timestamp":"2014-06-04T21:06:22.537Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 TRACE ceilometer.compute.pollsters.bandwidth StatementError: Can't reconnect until invalid transaction is rolled back (original cause: InvalidRequestError: Can't reconnect until invalid transaction is rolled back) 'SELECT ports.tenant_id AS ports_tenant_id, ports.id AS ports_id, ports.name AS ports_name, ports.network_id AS ports_network_id, ports.mac_address AS ports_mac_address, ports.admin_state_up AS ports_admin_state_up, ports.status AS ports_status, ports.device_id AS ports_device_id, ports.device_owner AS ports_device_owner \\nFROM ports' [immutabledict({})]","@version":"1","@timestamp":"2014-06-04T21:06:22.537Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 14:06:21.865 18971 TRACE ceilometer.compute.pollsters.bandwidth ","@version":"1","@timestamp":"2014-06-04T21:06:22.537Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
{"message":"2014-06-04 21:06:21.661+0000: 2562: error : virExec:418 : Cannot find 'pm-is-supported' in path: No such file or directory","@version":"1","@timestamp":"2014-06-04T21:06:22.538Z","host":"bd-3-12","path":"/var/log/libvirt/libvirtd.log"}
{"message":"2014-06-04 21:06:21.661+0000: 2562: warning : virQEMUCapsInit:896 : Failed to get host power management capabilities","@version":"1","@timestamp":"2014-06-04T21:06:22.539Z","host":"bd-3-12","path":"/var/log/libvirt/libvirtd.log"}
{"message":"2014-06-04 21:06:21.696+0000: 2562: error : virExec:418 : Cannot find 'pm-is-supported' in path: No such file or directory","@version":"1","@timestamp":"2014-06-04T21:06:22.539Z","host":"bd-3-12","path":"/var/log/libvirt/libvirtd.log"}
{"message":"2014-06-04 21:06:21.696+0000: 2562: warning : virQEMUCapsInit:896 : Failed to get host power management capabilities","@version":"1","@timestamp":"2014-06-04T21:06:22.539Z","host":"bd-3-12","path":"/var/log/libvirt/libvirtd.log"}
{"message":"2014-06-04 21:06:21.770+0000: 2564: error : virExec:418 : Cannot find 'pm-is-supported' in path: No such file or directory","@version":"1","@timestamp":"2014-06-04T21:06:22.539Z","host":"bd-3-12","path":"/var/log/libvirt/libvirtd.log"}
{"message":"2014-06-04 21:06:21.770+0000: 2564: warning : virQEMUCapsInit:896 : Failed to get host power management capabilities","@version":"1","@timestamp":"2014-06-04T21:06:22.539Z","host":"bd-3-12","path":"/var/log/libvirt/libvirtd.log"}
{"message":"2014-06-04 21:06:21.814+0000: 2563: error : virExec:418 : Cannot find 'pm-is-supported' in path: No such file or directory","@version":"1","@timestamp":"2014-06-04T21:06:22.539Z","host":"bd-3-12","path":"/var/log/libvirt/libvirtd.log"}
{"message":"2014-06-04 21:06:21.814+0000: 2563: warning : virQEMUCapsInit:896 : Failed to get host power management capabilities","@version":"1","@timestamp":"2014-06-04T21:06:22.539Z","host":"bd-3-12","path":"/var/log/libvirt/libvirtd.log"}
{"message":"2014-06-04 21:06:21.852+0000: 2563: error : virExec:418 : Cannot find 'pm-is-supported' in path: No such file or directory","@version":"1","@timestamp":"2014-06-04T21:06:22.539Z","host":"bd-3-12","path":"/var/log/libvirt/libvirtd.log"}
{"message":"2014-06-04 21:06:21.852+0000: 2563: warning : virQEMUCapsInit:896 : Failed to get host power management capabilities","@version":"1","@timestamp":"2014-06-04T21:06:22.539Z","host":"bd-3-12","path":"/var/log/libvirt/libvirtd.log"}
{"message":"2014-06-04 21:06:21.903+0000: 2565: error : virExec:418 : Cannot find 'pm-is-supported' in path: No such file or directory","@version":"1","@timestamp":"2014-06-04T21:06:22.539Z","host":"bd-3-12","path":"/var/log/libvirt/libvirtd.log"}
{"message":"2014-06-04 21:06:21.903+0000: 2565: warning : virQEMUCapsInit:896 : Failed to get host power management capabilities","@version":"1","@timestamp":"2014-06-04T21:06:22.539Z","host":"bd-3-12","path":"/var/log/libvirt/libvirtd.log"}
{"message":"Jun  4 14:06:23 bd-3-12 sudo:  neutron : TTY=unknown ; PWD=/ ; USER=root ; COMMAND=/usr/bin/neutron-rootwrap /etc/neutron/rootwrap.conf ovs-vsctl --timeout=2 list-ports br-int","@version":"1","@timestamp":"2014-06-04T21:06:23.541Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:23 bd-3-12 sudo: pam_unix(sudo:session): session opened for user root by (uid=110)","@version":"1","@timestamp":"2014-06-04T21:06:23.541Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:23 bd-3-12 sudo: pam_unix(sudo:session): session closed for user root","@version":"1","@timestamp":"2014-06-04T21:06:23.541Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:23 bd-3-12 sudo:  neutron : TTY=unknown ; PWD=/ ; USER=root ; COMMAND=/usr/bin/neutron-rootwrap /etc/neutron/rootwrap.conf ovs-vsctl --timeout=2 --format=json -- --columns=name,external_ids list Interface","@version":"1","@timestamp":"2014-06-04T21:06:23.541Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:23 bd-3-12 sudo: pam_unix(sudo:session): session opened for user root by (uid=110)","@version":"1","@timestamp":"2014-06-04T21:06:23.541Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:23 bd-3-12 sudo: pam_unix(sudo:session): session closed for user root","@version":"1","@timestamp":"2014-06-04T21:06:23.541Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:25 bd-3-12 sudo:  neutron : TTY=unknown ; PWD=/ ; USER=root ; COMMAND=/usr/bin/neutron-rootwrap /etc/neutron/rootwrap.conf ovs-vsctl --timeout=2 list-ports br-int","@version":"1","@timestamp":"2014-06-04T21:06:25.545Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:25 bd-3-12 sudo: pam_unix(sudo:session): session opened for user root by (uid=110)","@version":"1","@timestamp":"2014-06-04T21:06:25.545Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:25 bd-3-12 sudo: pam_unix(sudo:session): session closed for user root","@version":"1","@timestamp":"2014-06-04T21:06:25.546Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:25 bd-3-12 sudo:  neutron : TTY=unknown ; PWD=/ ; USER=root ; COMMAND=/usr/bin/neutron-rootwrap /etc/neutron/rootwrap.conf ovs-vsctl --timeout=2 --format=json -- --columns=name,external_ids list Interface","@version":"1","@timestamp":"2014-06-04T21:06:25.546Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:25 bd-3-12 sudo: pam_unix(sudo:session): session opened for user root by (uid=110)","@version":"1","@timestamp":"2014-06-04T21:06:25.546Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:25 bd-3-12 sudo: pam_unix(sudo:session): session closed for user root","@version":"1","@timestamp":"2014-06-04T21:06:25.546Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:27 bd-3-12 sudo:  neutron : TTY=unknown ; PWD=/ ; USER=root ; COMMAND=/usr/bin/neutron-rootwrap /etc/neutron/rootwrap.conf ovs-vsctl --timeout=2 list-ports br-int","@version":"1","@timestamp":"2014-06-04T21:06:27.575Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:27 bd-3-12 sudo: pam_unix(sudo:session): session opened for user root by (uid=110)","@version":"1","@timestamp":"2014-06-04T21:06:27.575Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:27 bd-3-12 sudo: pam_unix(sudo:session): session closed for user root","@version":"1","@timestamp":"2014-06-04T21:06:27.575Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:27 bd-3-12 sudo:  neutron : TTY=unknown ; PWD=/ ; USER=root ; COMMAND=/usr/bin/neutron-rootwrap /etc/neutron/rootwrap.conf ovs-vsctl --timeout=2 --format=json -- --columns=name,external_ids list Interface","@version":"1","@timestamp":"2014-06-04T21:06:27.575Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:27 bd-3-12 sudo: pam_unix(sudo:session): session opened for user root by (uid=110)","@version":"1","@timestamp":"2014-06-04T21:06:27.575Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"Jun  4 14:06:27 bd-3-12 sudo: pam_unix(sudo:session): session closed for user root","@version":"1","@timestamp":"2014-06-04T21:06:27.575Z","host":"bd-3-12","path":"/var/log/auth.log"}
{"message":"2014-06-04 14:06:26.824 18971 WARNING ceilometer.compute.pollsters.bandwidth [-] Ignoring instance instance-00000d1d: Can't reconnect until invalid transaction is rolled back (original cause: InvalidRequestError: Can't reconnect until invalid transaction is rolled back) 'SELECT ports.tenant_id AS ports_tenant_id, ports.id AS ports_id, ports.name AS ports_name, ports.network_id AS ports_network_id, ports.mac_address AS ports_mac_address, ports.admin_state_up AS ports_admin_state_up, ports.status AS ports_status, ports.device_id AS ports_device_id, ports.device_owner AS ports_device_owner \\nFROM ports' [immutabledict({})]","@version":"1","@timestamp":"2014-06-04T21:06:27.576Z","host":"bd-3-12","path":"/var/log/ceilometer/ceilometer-agent-compute.log"}
